lib/rubocop/cop/layout/ordered_methods.rb

```diff
@@ -133,12 +130,29 @@ def group_methods_by_access_modifier(nodes)
           end
         end
 
+        def self.method_name(node)
+          return node.method_name unless node.send_type?
+
+          node.first_argument.method_name
+        end
+
         def ordered?(left_method, right_method)
           comparison = COMPARISONS[cop_config['EnforcedStyle']]
           raise Error, ERR_INVALID_COMPARISON if comparison.nil?
 
           comparison.call(left_method, right_method)
         end
+
+        def relevant_node?(node)
+          (node.defs_type? || node.def_type?) && !ignored_method?(node.method_name)
```

Comment:
Did you mean to use the extracted `.method_name` to get the node's method name here and on line 153?

Reply:
Not really necessary, since it first checkk that the node is either `def` or `defs`, and the `method_name` handles send nodes, in order to get the defined method name, and not the name of the macro (e.g. get `foo` and not `memoize` in case of `memoize def foo`.
Though since it will return the same (node.method_name) for a node that is not of a send type, I can call it. I'd rather keep it as is, as `method_name` is defined on the class level

Reply:
Ah, I see. Thanks!